covidapp/ios/Podfile

```diff
@@ -0,0 +1,46 @@
+platform :ios, '10.0'
```

Comment:
1% of users are still on iOS 9.0
Since this project is meant to support global number of users, it would be nice to make iOS 9.0 the minimum.... but I'll relax on that if there are great technical benefits to a higher minimum:

https://david-smith.org/iosversionstats/
https://developer.apple.com/support/app-store/

Reply:
Maybe use whatever is the minimum for React-Native

Reply:
React-Native requires iOS 10.0, so stick with that:
https://github.com/facebook/react-native#-requirements

Reply:
https://github.com/facebook/react-native#-requirements.

I think the minimum is iOS 10

Reply:
This is a very good point.  Need to dig in more - I believe EXPO on react native does not work on iOS < 10.  @harperreed as well